pkg/agent/cluster.go

```diff
@@ -186,8 +188,9 @@ func (czero *Cluster) IsBootstrapComplete() (bool, error) {
 			return false, errors.New("cluster metadata returned nil from Agent Rest API")
 		}
 
-		if !checkHostsValidations(clusterMetadata, logrus.StandardLogger()) {
-			return false, errors.New("cluster host validations failed")
+		validationsErr := checkValidations(clusterMetadata, czero.installHistory.ValidationHistory, logrus.StandardLogger())
+		if validationsErr != nil {
+			return false, errors.Wrap(validationsErr, "cluster host validations failed")
```

Comment:
Previously `checkHostsValidations()` returned false (and we bailed out of this function) if _any_ validations were not passing. In practice I suspect this may have meant that we rarely get past this point, since _something_ is usually complaining.
With this change, we no longer return here unless there is an actual error. That's probably the right thing to do, but it results in dozens of warning messages in the logs saying `Cluster has stopped installing... working to recover installation` (from line 204).
Not sure of the right thing to do. It might be to just not log that warning.

Reply:
Perhaps we don't need this message anymore now that we are printing the validations.